app/controllers/teams_controller.rb

```diff
@@ -53,6 +53,6 @@ def set_team
 
     # Only allow a trusted parameter "white list" through.
     def team_params
-      params.require(:team).permit(:name)
+      params.require(:team).permit(:name, :private, :description, :kindofsport)
```

Comment:
Please use Snake case (`kind_of_sport`) throughout all variable and field names.